### PR TITLE
Add Spiced files

### PR DESCRIPTION
**Pull request by mgczarnievicz**

### CODE REVIEW

File: SP_README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+
+# Student Recap [cohort-Name]-web--[year]
+
+> Please review and select the files required for the specific cohort (NF or SP). Delete any files that are irrelevant to the respective cohort. For the files you choose to keep:
+
+- If the file is for SP, remove all mentions of NF.
+- If the file is for NF, remove all mentions of SP.
+>
```
Comment (ahohnsen) [inline SP_README.md:8]: If I understand correctly, we will have two Readme files in this repository - one for NF and one for SP. Maybe we should add these lines to both?!

File: SP_README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+| afternoon | **Recap Project 4** | **Recap Project 4** | **Project Presentation** | Self Directed Learning | Next.js Dynamic Routes     |
+
+### week 7
+
+|           | Monday                  | Tuesday                 | Wednesday              | Thursday            | Friday              |
+|-----------|-------------------------|-------------------------|------------------------|---------------------|---------------------|
+| morning   | React Styled Components | React Component Testing | React Global State     | React Data Fetching | **Recap Project 5** |
+| afternoon | JS Unit Testing         | React Global State      | Self Directed Learning | **Recap Project 5** | **Recap Project 5** |
+
+### week 8
```
[the remark this answers is not in the document]
Comment (ahohnsen) [reply SP_README.md:76]: This should be capital, as the other weeks. 🤔

File: docs/deployment-vercel.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ Vercel is a popular cloud web hosting provider with a fast deployment workflow.
    - After deployment is complete, you’ll receive a URL to visit your live website.
    - Vercel provides automatic updates: Every time you push changes to GitHub, your site will redeploy.
    - When you create a pull request on Github, Vercel will automatcally create a **preview deployment** for that feature branch and post a link in the respective PR.
-6. **Deployment Protection**     
+6. **Deployment Protection**
```
Comment (ahohnsen) [inline docs/deployment-vercel.md:22]: I actually added the spaces here, to move the following text to the next line (see preview).

File: docs/install-manual-en.md [hunk 4]
```diff
@@ -2,12 +2,20 @@
...
-```
-cd neuefische
+```bash
+cd web-bootcamp
 ```
 
 Clone the repository via its SSH URL:
 
-```
+NeueFisher
```
Comment (ahohnsen) [inline docs/install-manual-en.md:11]:
```suggestion
NeueFische
```

**Review by ahohnsen — APPROVED** [review]
Hey there,

I left a few comments.

Perhaps it would be helpful to create a small checklist or guide for all coaches to ensure they know exactly what needs to be adjusted before the course begins. Otherwise, it becomes necessary to manually open every file and check if NF or SP is mentioned anywhere (e.g., the Install Guide, where there are commands for both SP and NF, and the unnecessary one needs to be deleted).

**Comment by mgczarnievicz** [comment]
> Perhaps it would be helpful to create a small checklist or guide for all coaches to ensure they know exactly what needs to be adjusted before the course begins. Otherwise, it becomes necessary to manually open every file and check if NF or SP is mentioned anywhere (e.g., the Install Guide, where there are commands for both SP and NF, and the unnecessary one needs to be deleted).

We can add it in the internal one, that has the instructions of the cohort setup no? And we can update it (I think is not up to date, I may be wrong)

**Pull request merged**